app/javascript/components/search/results/StudySearchResult.jsx

```diff
@@ -212,6 +215,16 @@ export function logSelectSearchResult(study, logProps={}) {
   // This is needed to easily analyze changes in click-through rate (CTR).
   logProps.numSearches = numSearches
 
+  // Number of selections done for the current search.  Users can click
+  // multiple results for a given search, e.g. via "Open in new tab", and this
+  // property lets us account for that when analyzing CTR and related  metrics.
+  if (numSearches != prevNumSearches) {
```

Comment:
Let's use the strict equals check here:
```suggestion
  if (numSearches !== prevNumSearches) {
```